dev/sbt-checkstyle

```diff
@@ -17,17 +17,13 @@
 # limitations under the License.
 #
 
+build_profiles=$1
+echo "Checking Spark style using SBT with these arguments: ${build_profiles}"
+
 # NOTE: echo "q" is needed because SBT prompts the user for input on encountering a build file
 # with failure (either resolution or compilation); the "q" makes SBT quit.
 ERRORS=$(echo -e "q\n" \
-    | build/sbt \
-        -Pkinesis-asl \
-        -Pmesos \
-        -Pkubernetes \
-        -Pyarn \
-        -Phive \
-        -Phive-thriftserver \
-        checkstyle test:checkstyle \
```

Comment:
@wangyum, can we keep this existing profiles if arguments are not given? This script can be manually ran as well. This is intentionally matched with `lint-java`'s.

Reply:
I think that is what the `${1:-...}` syntax does